deploy/database/views.game.sql

```diff
@@ -22,6 +22,7 @@ AS SELECT
     p.name_ingame AS player_name,
     p.autopass,
     p.fire_overshooting,
+    (length(p.vacation_message) > 0) as is_on_vacation,
```

Comment:
Something i'll be looking for in testing - since there's no separate option for *whether* a player is on vacation other than the message itself, is it easy for a player to remove their vacation message and see that they've done so?  It'd be frustrating if you left some whitespace in the box and were still on vacation and couldn't figure out why.